pkg/aspectsupport/denyChecker/config/config.proto

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package pkg.adaptersupport.denyChecker.config;
+
```

Comment:
non-blocker: Should we set an `option go_package="config";` in these proto files?  And if so, can we rename the core message to be Params (or similar) to match the layout of the yaml and to avoid config.Config stutter?

this would allow us to simply:
`import "istio.io/mixer/pkg/aspectsupport/denyChecker/config"` instead of:
`import pb "istio.io/mixer/pkg/aspectsupport/denyChecker/config"`